scripts/configure-cluster-with-encryption.sh

```diff
@@ -4,11 +4,11 @@
 
 # this script downloads all tools required to use FLE with mongodb, then starts a cluster of the provided configuration (sharded on 8.0 server)
 
-export CWD=$(pwd);
-export DRIVERS_TOOLS_PINNED_COMMIT=35d0592c76f4f3d25a5607895eb21b491dd52543;
+export CWD=$(pwd)
+export DRIVERS_TOOLS_PINNED_COMMIT=4e18803c074231ec9fc3ace8f966e2c49d9874bb
```

Comment:
This is the latest commit as of yesterday ~noon.  I have another ticket in this epic to better figure out how to integrate with drivers-evergreen-tools so it is maintainable for mongoose developers, but I haven't started it yet.  So for now, I'm sticking with this script.